packages/shared/components/Button.svelte

```diff
@@ -68,7 +69,7 @@
     class:is-busy={isBusy}
     style:--border-width={outline ? '1px' : '0px'}
     use:bindEvents={events}
-    on:click|stopPropagation={onClick}
+    on:click|stopPropagation={isDebounceEnabled ? debounce(onClick) : onClick}
```

Comment:
Remove the conditional, but add a wait value of `100` ms (maybe even `50`?) to the `debounce` function. It feels very laggy with the default value of 500ms.